codegenerator/cli/src/config_parsing/human_config.rs

```diff
@@ -542,6 +542,16 @@ pub mod fuel {
             description = "Index Burn receipts. The option can be omitted when the event name is Burn."
         )]
         pub burn: Option<bool>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        #[schemars(
+            description = "Index Call receipts. The option can be omitted when the event name is Call."
+        )]
+        pub call: Option<bool>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        #[schemars(
+            description = "Index TransferOut receipts. The option can be omitted when the event name is TransferOut."
+        )]
+        pub transfer_out: Option<bool>,
```

Comment:
I'm not loving all these single option flags. Why don't we have an optional field called "receipt_type" or something which can be an enum for Mint,  Burn, Call, TransferOut? Especially since it's currently just a way to have an alias.

Or you could even make it an enum with log_id.

Reply:
That was a good idea. I've decided to add the `type` field.